### ISSUE TITLE
Search results page: cannot close search criteria box using keyboard

### ISSUE THREAD

**Issue by AlOneill** [FreeUKGen/FreeBMD2#475]
It is possible to tab to the Close neither-button-nor-link in the "When you searched for:" box, but not to operate it. Enter should close it.

If it were marked up as a `button`, then Enter or Space should close it. 

If it were marked up as a link, which would be wrong because it does not have an `href`, then Enter should close it (but not Space). It is marked up using a `div` with no semantics.

Another tab moves the focus on, so at least it is not a keyboard trap!

**Comment by PatReynolds** [comment]
Has to be implemented as a button. Enter and space should operate it. Enter is the most important.

**Comment by Vino-S** [comment]
Ready for testing in test3

**Comment by AlOneill** [comment]
Can now close with either Space or Enter. Thank you, @Vino-S 

Did notice though that if one uses 'Tools' to see the full 'About this query' info, there is no option to go back to results, so had to 'Revise search' and repeat the search.

**Comment by Vino-S** [comment]
Thanks for testing @AlOneill 

_Did notice though that if one uses 'Tools' to see the full 'About this query' info, there is no option to go back to results, so had to 'Revise search' and repeat the search._

We have a back button in 'About this query' page

**Comment by AlOneill** [comment]
Oops, missed that! Apologies, @Vino-S
